lib/Dialect/P4HIR/P4HIR_Ops.cpp

```diff
@@ -1513,7 +1603,8 @@ void P4HIR::ParserOp::print(mlir::OpAsmPrinter &printer) {
     }
 
     printer << ' ';
-    printer.printRegion(getRegion(), /*printEntryBlockArgs=*/false, /*printBlockTerminators=*/true);
+    printer.printRegion(getRegion(), /*printEntryBlockArgs=*/false,
```

Comment:
Are these changes from clang-format? I think it would fold back to the original line.